planning/jet/jetsim.cc

```diff
@@ -202,10 +227,27 @@ void go() {
     accum_geo->flush();
 
     const cv::Mat image = camera->extract_image();
-    if (SHOW_CAMERA) {
-      cv::imshow("Localization Camera", image);
-      cv::waitKey(1);
+
+    {
+      const cv::Mat localization_camera_image = camera->extract_image();
+      cv::Mat localization_camera_image_rgb;
+      cv::cvtColor(localization_camera_image, localization_camera_image_rgb,
+                   cv::COLOR_GRAY2BGR);
+      auto world_from_opengl_camera = world_from_jet * jet_from_camera;
+      auto world_from_markers = estimation::vision::get_world_from_marker_centers(
+          localization_camera_image_rgb, world_from_opengl_camera);
```

Comment:
Factor this into a function (Jetsim main loop ought not to convert image types etc)